tests/emukit/core/optimization/test_context_manager.py

```diff
@@ -28,3 +27,27 @@ def test_context_manager_expand_vector(space, context):
     x_expanded = context_manager.expand_vector(x)
     assert x_expanded.ndim == 2
     assert np.array_equal(x_expanded, np.array([[0.3, 0.5]]))
+
+
+@pytest.fixture
+def catg_space():
+    return ParameterSpace([ContinuousParameter('x1', 0, 15), 
+                           CategoricalParameter('x2', OneHotEncoding([0, 1, 2, 3])),
+                           CategoricalParameter('x3', OneHotEncoding([1, 2, 3, 4, 5])),
+                           ContinuousParameter('x4', -2, 3)]) 
+
+@pytest.fixture
```

Comment:
Are these utilities (catg_space, catg_context) defined as fixtures to indicate their reusability across tests? If not, would it not make more sense to initialize them inside test_context_manager_catg_expand_vector?

Reply:
I made them fixtures in case more tests with multiple context variables need to be added, but can definitely change them to parameters, let me know!